python/ray/autoscaler/ray-schema.json

```diff
@@ -301,18 +280,6 @@
             "type": "string",
             "description": "If using multiple node types, specifies the head node type."
         },
-        "worker_default_node_type": {
-            "type": "string",
-            "description": "DEPRECATED."
-        },
-        "head_node": {
-            "type": "object",
```

Comment:
hmm looks this hasn't been deprecated previously?

Reply:
Looks like we didn't mark it as such in the schema, but we have been logging deprecation warnings for this field and worker_nodes for over a year.
https://github.com/ray-project/ray/blame/c7ae787cc8468d8087aa3483b472f40624bd4a22/python/ray/autoscaler/_private/util.py#L189-L190